Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/NewMeterWizard/AssetPage.tsx

```diff
@@ -536,7 +567,39 @@ export default function AssetPage(props: IProps) {
                                 </div>
                             </div> : null}
                         </div>
-                    </Modal>
+                </Modal>
+
+                <Modal Show={showDrawings} Title={'Drawings'} ShowX={true} Size={'lg'} CallBack={() => setShowDrawings(false)}
+                    ShowCancel={false} ConfirmText={'Done'}
+                >
+                    <div className="row">
+                        <div className="col" style={{ width: '100%' }}>
+                            <Table<SystemCenter.Types.LocationDrawing>
+                                cols={[
+                                    { key: 'Name', field: 'Name', label: 'Name', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
+                                    { key: 'Description', field: 'Description', label: 'Description', headerStyle: { width: 'auto' }, rowStyle: { width: 'auto' } },
+                                ]}
+                                tableClass="table table-hover"
+                                data={drawingData}
+                                sortKey={drawingSortKey}
+                                ascending={drawingAscending}
+                                onSort={(d) => {
+                                    dispatch(LocationDrawingSlice.Sort({ SortField: d.colField, Ascending: d.ascending }));
+                                }}
+                                onClick={(d) => {
+                                    window.open(d.row.Link, '_blank');
+                                }}
+                                theadStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
+                                tbodyStyle={{ display: 'block', overflowY: 'scroll', maxHeight: '400px', width: '100%' }}
+                                rowStyle={{ fontSize: 'smaller', display: 'table', tableLayout: 'fixed', width: '100%' }}
+                            />
+                        </div>
+                    </div>
+                </Modal>
+
+                <ToolTip Show={hover === 'drawings' && (props.Location == null || props.Location.ID == null || props.Location.ID == 0 || drawingData.length == 0)} Theme={'dark'} Position={'top'} Target={'drawings'} Zindex={9999}>
+                    <p>No drawings associated with this substation.</p>
+                </ToolTip>
```

Comment:
Seems like that code matches the one in `MeterLocationProperties` so maybe we can move it into a separate component?